utils/types.ts

```diff
@@ -0,0 +1,2 @@
+export const objectKeys = <Type extends object>(value: Type) =>
```

Comment:
oh you could increase coverage by either ignoring this file or adding a unit test for it.

Reply:
Do you typically ignore a file like that, or is the convention to create a test for it?

Reply:
I mean it's a pretty basic function created solely for a typing DX improvement. I'm not very worried about it.